### PR TITLE
[OKTA-802677] Updates to SCIM 2.0 with entitlements

### PR DESCRIPTION
**Pull request by annejuan-okta**
## Description:
- **What's changed?**  Description added to examples, also added more info about schemas
- **Is this PR related to a Monolith release?**  Nope

### Resolves:

* [OKTA-802677](https://oktainc.atlassian.net/browse/OKTA-802677)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md](https://github.com/okta/okta-developer-docs/blob/f4e14c1eb7b4a213ac93ee5cfac3d016ad7c26b6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-with-entitlements%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2c16537f-b068-403a-8ccf-f9e607d81bc1) | 84  | 94  | 70  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.4fc06962-a68b-447d-875d-cd7410474a8d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md](https://github.com/okta/okta-developer-docs/blob/5ba5ab7ec8217059747e72f7e492a4757ab73580/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-with-entitlements%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/c5bf39f3-566c-4ddf-acd9-ef9f6b0c3b97) | 84  | 94  | 70  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.f6a08bf6-619b-4d7b-9c05-ff1461d97e94?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by annejuan-okta** [comment]
Scott approved the content in Slack.

File: packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md [hunk 1]
```diff
@@ -17,8 +17,9 @@ This guide teaches you how to create a SCIM 2.0 server that supports provisionin
 
 #### What you need
 
-* [Okta Developer Edition organization](https://developer.okta.com/signup).
-* Some basic development experience with the System for Cross-domain Identity Management (SCIM) [core schema](https://datatracker.ietf.org/doc/html/rfc7643) and [protocol](https://datatracker.ietf.org/doc/html/rfc7644)
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md:20]:
```suggestion
* [Okta Developer Edition org](https://developer.okta.com/signup)
```

File: packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md [hunk 2]
```diff
@@ -17,8 +17,9 @@ This guide teaches you how to create a SCIM 2.0 server that supports provisionin
...
+* Some basic development experience with the SCIM [core schema](https://datatracker.ietf.org/doc/html/rfc7643) and [protocol](https://datatracker.ietf.org/doc/html/rfc7644)
+* [Okta Identity Governance](https://help.okta.com/okta_help.htm?type=oie&id=ext-iga) in order to use [entitlements](https://help.okta.com/okta_help.htm?type=oie&id=ext-entitlement-mgt)
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md:22]:
```suggestion
* [Okta Identity Governance](https://help.okta.com/okta_help.htm?type=oie&id=ext-iga) to use [entitlements](https://help.okta.com/okta_help.htm?type=oie&id=ext-entitlement-mgt)
```

File: packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md [hunk 3]
```diff
@@ -93,10 +95,35 @@ For any required custom schema extensions, implement the `/Schemas` endpoint. Se
...
+* `User:Enterprise`
+
+#### Entitlements schema
+
+The `urn:okta:scim:schemas:core:1.0:Entitlement` schema contains the following information:
+
+|Parameter | Type | Description | Notes |
+|---|---|---|---|
+| `id` | String | The ID of the entitlement | Required |
+| `displayName` | String | The name displayed in the UI for the entitlement | Required|
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md:110]:
```suggestion
| `displayName` | String | The name that appears in the Admin Console for the entitlement | Required|
```

File: packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md [hunk 4]
```diff
@@ -93,10 +95,35 @@ For any required custom schema extensions, implement the `/Schemas` endpoint. Se
...
+| `type` | String | Corresponds with the `ResourceType` name field for the entitlement, such as "License" or "Permission set" | Required |
+| `description` | String | A human-readable description of the entitlement. This appears in the Governance tab. | Optional, max length 1000 characters |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md:112]:
```suggestion
| `description` | String | A human-readable description of the entitlement. This appears in the **Governance** tab. | Optional, max length 1000 characters |
```

File: packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md [hunk 5]
```diff
@@ -93,10 +95,35 @@ For any required custom schema extensions, implement the `/Schemas` endpoint. Se
...
+See [Custom entitlement with extensions](#custom-entitlement-with-extensions) for an example schema.
+
+#### Roles schema
+
+The `urn:okta:scim:schemas:core:1.0:Entitlement` schema contains the following information:
+
+|Parameter | Type | Description | Notes |
+|---|---|---|---|
+| `id` | String | The ID of the role | Required |
+| `displayName` | String | The name displayed in the UI for the role | Required|
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md:123]:
```suggestion
| `displayName` | String | The name that appears in the Admin Console for the role | Required|
```

**Review by susanharper-okta — COMMENTED** [review]
edits and a comment - looks like the release notes file tagged along in your PR

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md](https://github.com/okta/okta-developer-docs/blob/0200299153c5b123dfb0b293afd2fe65878e82c7/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-with-entitlements%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/8bc85ba4-77c1-477c-9efd-71e0d789285c) | 84  | 94  | 70  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.de0a84e6-ba18-40c3-8c6a-b049c2f82f22?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/scim-with-entitlements/main/index.md](https://github.com/okta/okta-developer-docs/blob/b09131b56bf766bb9fecb3c705d591bfcce5f7cb/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fscim-with-entitlements%2Fmain%2Findex.md) | [85](https://okta.acrolinx.cloud/api/v1/checking/scorecards/478fc6e3-f5e1-4cba-bb38-136d814846e2) | 84  | 94  | 74  | 100  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.1bad5360-f3a6-4ca6-be9d-67e68e37c9a9?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by susanharper-okta — COMMENTED** [review]
LGTM

**Review by susanharper-okta — APPROVED** [review]

**Pull request merged**